### ISSUE TITLE
Translation window is always 1-line size

### ISSUE THREAD

**Issue by Badiboy** [WeblateOrg/weblate#4157]
**Describe the bug**

The translation editing window always appears in 1 line size, despite the text has multiple lines.
![image](https://user-images.githubusercontent.com/5613421/87148352-ec267080-c2b6-11ea-9719-6b492240450a.png)
Nearly always, very rare in some unknown surcomstances it appears in normal size.

**To Reproduce**

Just go to the Project / Component / All strings

**Expected behavior**

This is the original text size:
![image](https://user-images.githubusercontent.com/5613421/87148491-26900d80-c2b7-11ea-8b51-3390bd53979e.png)

**Server configuration and status**

**Weblate deploy checks**

Server: Weblate 4.1.1 / Ububtu / DEBUG enabled
User: Windows 10 / Google Chrome (latest)

**Comment by nijel** [comment]
It is supposed to autoscale based on text length. I recall there were some fixes recently, but I can't find that right now.

Can you reproduce it on Hosted Weblate as well? Try this: https://hosted.weblate.org/translate/phpmyadmin/documentation/fr/?checksum=14445d0b432452ab

**Comment by Badiboy** [comment]
Looks fine!

![image](https://user-images.githubusercontent.com/5613421/87154920-b7b8b180-c2c2-11ea-98fa-26a228eb779d.png)

**Comment by Badiboy** [comment]
Hmmm. It seems that there are some problems with cache / cookies. 

I made a reload (Ctrl + F5) and window size become normale in the opened window AND if I go to the next messages. So ater Ctrl+F5 it starts working normally. Even if I close / open the window.

And I faced more issues with Ctrl+F5.
1. When I open the main site page I see only unformatted text without any style and images. Ctrl+F5 make it fine. After that it works fine for SOME time (I did not find what time should pass before it happens or what can lead to it).
2. When I click buttons like "Dismiss" in Warnings they bring me to "Not found" or error like that, I did not recorded it. After Ctrl+F5 they work fine.

These issues started appearing from 4.1, I never seen them before.

**Comment by nijel** [comment]
Did you run `collectstatic` after upgrading to 4.1? See https://docs.weblate.org/en/latest/admin/upgrade.html

**Comment by Badiboy** [comment]
Sure, my upgarde .sh is written according to this manual and inlcudes it also.

I rerun it now and got:

> 0 static files copied to '/drive/weblate-env/lib/python3.6/site-packages/data/static', 382 unmodified.

So seems that it's fine.

**Comment by nijel** [comment]
Do you run Weblate in debug mode? The JavaScript serving in that case is slightly different and might lead to caching issues.

**Comment by Badiboy** [comment]
Yes, DEBUG. I mentioned.

> Weblate deploy checks
> Server: Weblate 4.1.1 / Ububtu / DEBUG enabled

_PS: I failed to run without DEBUG: static files are failed to be found. Cannot solve it. :(_

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/admin/install.html#running-web-server

**Comment by Badiboy** [comment]
Thank you, now I disabled DEBUG :)

Without DEBUG I see adequate window sizes. So that was a problem within DEBUG mode.

**Comment by nijel** [comment]
Did you customize configuration anyhow or using default one? Most importantly did you enable COMPRESS_ENABLED?

**Comment by nijel** [comment]
Anyway in DEBUG mode, the URLs for CSS/JS files should include git revision to avoid caching issues. Do you see these?

**Comment by Badiboy** [comment]
> Did you customize configuration anyhow or using default one? Most importantly did you enable COMPRESS_ENABLED?
Generally most of the settinfgs are default. I make a minimal customizatin of the config to fulfill base needs. Because it's a kind of "pet project", so as far as it start solving my needs I stopped customization. :) The only problem was with disabling DEBUG, but now it's gone, thanks to you.
COMPRESS_ENABLED - no, there was no such option in any description. I looked in the code and see now how it process cache settings.
